inc/options/general.php

```diff
@@ -494,11 +494,14 @@
 			);
 
 			$time_options = array();
+			$timezone     = new \DateTimeZone( get_user_meta( get_current_user_id(), 'timezone', true ) ?: wp_timezone()->getName() );
```

Comment:
Users don't have an option to choose a timezone different from the WP site timezone, so there is no "timezone" in user meta.  You can just set the variable to `wp_timezone()`.  If it's only used once below, just use `wp_timezone()` there.

Reply:
Updated